apps/web/src/styles/theme.ts

```diff
@@ -201,6 +219,9 @@ const theme = extendTheme({
         md: { height: "42px", minWidth: "42px", padding: "0 12px" },
         lg: { height: "48px", minWidth: "48px", padding: "0 12px" },
       },
+      defaultProps: {
+        size: "lg",
```

Comment:
not sure if it's a good thing. so, now we need to check all the places where the button is used and check we didn't break anything (and ideally, remove size='lg' in such places), if you did feel free to leave it as is

Reply:
I removed all size="lg" and double checked the existing buttons, they look fine